doc/man/man3/crypto_lock.3monocypher

```diff
@@ -82,9 +82,9 @@ See
 about random number generation (use your operating system's random
 number generator).
 .It Fa mac
-a 16-byte
-.Em message authentication code (MAC) ,
-that can only be produced by someone who knows the session key.
+A 16-byte
+.Em message authentication code
+(MAC), that can only be produced by someone who knows the session key.
```

Comment:
"message authentication code" in italics, (MAC) not. I think this is the normal way of doing it